File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.swift
func solution(_ phone_number:String) -> String {
    
    var mapping = phone_number.map {String($0)} //문자열을 배열로 바꿈
    for i in 0..<mapping.count - 4 {
        mapping[i] = "*"
    }
    
    return mapping.joined()
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.py
def solution(phone_number):
    answer = list(phone_number)
    answer.reverse()
    result = []

    result = answer[0:4]
    for i in (answer[4:]):
        temp = i.replace(i,'*')
        result.append(temp)

    result.reverse()
    return ''.join(result)